config/global.json

```diff
@@ -3232,5 +3235,125 @@
+                "munch==2.5.0",
+                "pillow==7.0.0",
+                "ffmpeg-python==0.2.0"
+            ],
+            "generate_method": {
+                "name": "generate",
+                "args": {
+                    "base": [
+                        "model_file",
+                        "output_path",
+                        "save_images",
+                        "num_samples"
+                    ],
+                    "custom": {
+                        "image_paths_input": [
+                            "models/00022_WGAN_CARDIAC_AGING/sample_image.png",
+                            "models/00022_WGAN_CARDIAC_AGING/sample_image.png",
+                            "models/00022_WGAN_CARDIAC_AGING/sample_image.png"
+                        ],
+                        "aging_input": [
+                            -4,
+                            10,
+                            2
+                        ],
+                        "data_type": "2d",
+                        "view": "la",
+                        "subcat": "2ch"
+                    }
+                }
+            }
+        },
+        "selection": {
+            "performance": {},
+            "use_cases": [
+                "classification",
+                "segmentation"
+            ],
+            "organ": [
+                "heart",
+                "chest"
+            ],
+            "modality": [
+                "MRI",
+                "Cardiac imaging",
+                "Cardiography",
+                "full-field digital"
+            ],
+            "vendors": [],
+            "centres": [],
+            "function": [
+                "image to image",
+                "image generation",
+                "data augmentation"
+            ],
+            "condition": [
+                "age"
+            ],
+            "dataset": [
+                "UK Biobank"
+            ],
+            "augmentations": [
+                "resize"
+            ],
+            "generates": [
+                "cardiac image",
+                "full-field digital"
+            ],
+            "height": 256,
+            "width": 256,
+            "depth": null,
+            "type": "pix2pix",
+            "license": null,
+            "dataset_type": "non-public",
+            "privacy_preservation": null,
+            "tags": [
+                "Cardiac imaging",
+                "pix2pix",
+                "Pix2Pix"
+            ],
+            "year": "2022"
+        },
+        "description": {
+            "title": "Generates cardiac images with age offset from real images (Trained on UK Biobank)",
+            "provided_date": "2022",
+            "trained_date": "2022",
+            "provided_after_epoch": 299,
+            "version": "0.0.1",
+            "publication": "https://www.frontiersin.org/articles/10.3389/fcvm.2022.983091",
+            "doi": [
+                ""
+            ],
+            "inputs": [
```

Comment:
Great that you specified the inputs here!! 💯  
If possible, could you also add this information to the [model documentation page](https://github.com/RichardObi/medigan/blob/feature/model_00022_cardiac/docs/source/model_documentation.md)? (see [here](https://medigan.readthedocs.io/en/latest/models.html#cyclegan-brain-mri-t1-t2) for an example)

https://github.com/RichardObi/medigan/blob/feature/model_00022_cardiac/docs/source/model_documentation.md